ansible_runner_service.py

```diff
@@ -156,13 +185,29 @@ def main(test_mode=False):
         app.config['WTF_CSRF_ENABLED'] = False
         return app.test_client()
 
-    # Start the API server
-    app.run(host=configuration.settings.ip_address,
-            port=configuration.settings.port,
-            threaded=True,
-            ssl_context=ssl_context,
-            debug=configuration.settings.debug,
-            use_reloader=False)
+    cancel = threading.Event()
+    cancel.clear()
+
+    t = threading.Thread(
+        target=cleanup_thread,
+        args=(cancel, configuration.settings.artifacts_remove_frequency * 60 * 60 * 24)
+    )
+    if configuration.settings.mode == 'prod':
+        t.start()
```

Comment:
I think we can move the `cancel` and `t` creation inside this `if`.

Reply:
yeah agree, my ide was screaming at me to make it a global variable